starlight/admin.py

```diff
@@ -0,0 +1,54 @@
+from django import forms
+from django.contrib import admin
+from django.contrib.auth.forms import ReadOnlyPasswordHashField
+
+from starlight.models import Skill, Competency, Employee
+
+admin.site.register(Skill)
+admin.site.register(Competency)
+
+class UserCreationForm(forms.ModelForm):
+    """A form for creating new users. Includes all the required
+    fields, plus a repeated password."""
+    password1 = forms.CharField(label='Password', widget=forms.PasswordInput)
+    password2 = forms.CharField(label='Password confirmation', widget=forms.PasswordInput)
+
+    class Meta:
+        model = Employee
+        fields = ('full_name', 'email', )
+
+    def clean_password2(self):
+        # Check that the two password entries match
+        password1 = self.cleaned_data.get("password1")
+        password2 = self.cleaned_data.get("password2")
+        if password1 and password2 and password1 != password2:
+            raise forms.ValidationError("Passwords don't match")
+        return password2
+
+    def save(self, commit=True):
+        # Save the provided password in hashed format
+        user = super().save(commit=False)
+        user.set_password(self.cleaned_data["password1"])
+        if commit:
+            user.save()
+        return user
+
+
+class UserChangeForm(forms.ModelForm):
+    """A form for updating users. Includes all the fields on
+    the user, but replaces the password field with admin's
+    password hash display field.
+    """
+    password = ReadOnlyPasswordHashField()
```

Comment:
I assume you will update the change form in another PR to allow the user to change their password? :)

Reply:
Yes, good idea